joss_paper/paper.md

```diff
@@ -424,6 +424,9 @@ aas-doi: Add DOI HERE
 aas-journal: The Astrophysical Journal
 ...
 
+[^*]: The author list in this paper is sorted by number of commits to the core
```

Comment:
```suggestion
[^*]: The author list in this paper is sorted by number of non-merge commits to the core
```
maybe?

Reply:
except that it includes merge commits as currently authored. I don't want to change the ordering on a whim as it will be a bit of work.

Reply:
Oh never mind me then.